ext/posix/tests/posix_ttyname_error_wrongparams.phpt

```diff
@@ -15,12 +15,10 @@ standard
 
 var_dump(posix_ttyname(0)); // param not a resource
 
-$descriptors = [["pty"], ["pty"], ["pty"], ["pipe", "w"]];
-$pipes = [];
-$process = proc_open('echo "foo";', $descriptors, $pipes);
+$bucket = stream_bucket_new(fopen('php://temp', 'w+'), '');
 
 try {
-    var_dump(posix_ttyname($process)); // wrong resource type
+    var_dump(posix_ttyname($bucket->bucket)); // wrong resource type
 } catch (TypeError $e) {
```

Comment:
Agree. This looks suspicious.
It looks like to keep compatibility, ``posix_ttyname()`` should be extended to support the ``Process`` objects or we need some common parent class or interface...

Reply:
This test was veryfing that invalid resources throw a TypeError, a process resource was never accepted by the ``posix_ttyname()`` function as it only accepts ``stream`` resources (or an ``int`` for a file descriptor).

Reply:
Oh, yes. My mistake.

Reply:
This is probably ok as proc wasn't exactly related here either. Better not to create zend_test dependency here because not everyone might want to have zend_test installed to be able to run posix tests. So I would prbably leave the stream bucket here.